DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -207,7 +207,7 @@ private int AddTabInternal(TabInfo tab, int afterTabId, int beforeTabId, bool in
             return tab.TabID;
         }
 
-        private void CreateLocalizedCopyInternal(TabInfo originalTab, Locale locale, bool allTabsModulesFromDefault, bool clearCache)
+        private void CreateLocalizedCopyInternal(TabInfo originalTab, Locale locale, bool allTabsModulesFromDefault, bool clearCache, bool insertAfterOriginal = false)

```

Comment:
I wonder if we actually need this `insertAfterOriginal` parameter. Do you see any use cases for not adding the translated page at the original position @dnnsoftware/approvers ?

I tested this a bit, and was only able to hit this `CreateLocalizedCopyInternal` method from lines 910 and 929 (the ones modified in this PR), but I wasn't able to hit the other ones, at lines 1083 and 1096.

For the one at 1083, I could only find references coming from deprecated code:
https://github.com/dnnsoftware/Dnn.Platform/blob/8321bdab6d9854a4417321f07532e8c414e8b7a3/DNN%20Platform/Library/Obsolete/TabController.cs#L64
https://github.com/dnnsoftware/Dnn.Platform/blob/8321bdab6d9854a4417321f07532e8c414e8b7a3/DNN%20Platform/Library/Obsolete/TabController.cs#L72

For the one at line 1096, I found this reference:
https://github.com/dnnsoftware/Dnn.Platform/blob/8321bdab6d9854a4417321f07532e8c414e8b7a3/DNN%20Platform/DotNetNuke.Web/UI/RibbonBarManager.cs#L363
which is used also by deprecated code:
https://github.com/dnnsoftware/Dnn.Platform/blob/8321bdab6d9854a4417321f07532e8c414e8b7a3/DNN%20Platform/DotNetNuke.Website.Deprecated/admin/ControlPanel/AddPage.ascx.cs#L94
Is it possible to see this control panel somehow in lastest versions?